src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,189 @@
+  status: boolean
+  uuid: string
+}
+
+const form = ref<Form>({
+  status: false,
+  uuid: ''
+})
+
+interface errorObject {
+  notificationTitle: string
+  notificationDescription: string
+  uuid: string
+}
+const objError: errorObject = {
+  notificationTitle: '',
+  notificationDescription: '',
+  uuid: ''
+}
+
+const isError = ref(false)
+const loading = ref(false)
+const saving = ref(false)
+const uuidRef = ref()
+const justCopied = ref(false)
+
+const error: Ref<errorObject> = ref({ ...objError })
+const errorLoadingConfiguration: Ref<errorObject> = ref({ ...objError })
+const errorSaving: Ref<errorObject> = ref({ ...objError })
+
+onMounted(() => {
+  getConfiguration()
+})
+
+async function getConfiguration() {
+  try {
+    loading.value = true
+    let getDataConfiguration = await ubusCall('ns.netifyd', 'status', {})
+    if (getDataConfiguration && getDataConfiguration.data) {
+      let configuration = getDataConfiguration.data
+      form.value.status = configuration.enabled
+      form.value.uuid = configuration.uuid
+    }
+  } catch (exception: any) {
+    isError.value = true
+    errorLoadingConfiguration.value.notificationTitle = t(
+      'error.cannot_retrieve_netify_informatics_configuration'
+    )
+    errorLoadingConfiguration.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  } finally {
+    loading.value = false
+  }
+}
+
+function clearErrors() {
+  error.value = {
+    notificationTitle: '',
+    notificationDescription: '',
+    uuid: ''
+  }
+}
+
+function handleSaveAutomaticToggle() {
+  clearErrors()
+  saving.value = true
+  const status = form.value.status ? 'enable' : 'disable'
+  ubusCall('ns.netifyd', status)
+    .then((response) => {
+      if (response.data && response.data.result && response.data.result === 'success') {
+        setTimeout(() => {
+          notificationsStore.createNotification({
+            title: t('standalone.netify_informatics.netify_informatics_' + status),
+            description: t(
+              'standalone.netify_informatics.netify_informatics_' + status + '_description'
+            ),
+            kind: 'success'
+          })
+        }, 500)
+      }
+    })
+    .catch((exception: AxiosError) => {
+      errorSaving.value.notificationTitle = t('error.cannot_save_configuration')
+      errorSaving.value.notificationDescription = t(getAxiosErrorMessage(exception))
+    })
+    .finally(() => (saving.value = false))
+}
+
+function openDashboard() {
+  window.open('https://portal.netify.ai/login', '_blank')
+}
+function copyUuid() {
+  navigator.clipboard.writeText(form.value.uuid)
+  justCopied.value = true
+  setTimeout(() => {
+    justCopied.value = false
+  }, 3000)
+}
+</script>
+
+<template>
+  <div>
+    <FormLayout class="max-w-6xl">
+      <template #description>
+        <p>{{ t('standalone.netify_informatics.content_description') }}</p>
+        <NeButton kind="tertiary" size="lg" @click="openDashboard()" class="ml-2 shrink-0">
```

Comment:
Let's fix vertical spacing and use `NeLink` instead (in general: NeButton is for actions, NeLink is for simple links). See https://github.com/NethServer/nethsecurity-ui/blob/main/src/components/standalone/backup_and_restore/MigrationContent.vue#L39-L44